src/thermompnn/proteinmpnn/model_utils.py

```
@@ -0,0 +1,1175 @@
+        self.num_positional_embeddings = num_positional_embeddings
+        self.side_chains = side_chains
+
+        if self.side_chains:
+            print('Side chain distances enabled!')
+            num_atom_combos = 5 * 14
+        else:
+            num_atom_combos = 25
+            print('Using only backbone distances!')
+
+        self.embeddings = PositionalEncodings(num_positional_embeddings)
+        node_in, edge_in = 6, num_positional_embeddings + num_rbf * num_atom_combos
```

Ruff (0.8.2)

Local variable `node_in` is assigned to but never used

Remove assignment to unused variable `node_in`

(F841)